eng/versioning.targets

```diff
@@ -83,7 +86,7 @@
       </AssemblyAttribute>
 
       <AssemblyAttribute Include="System.Reflection.AssemblyMetadata"
-                         Condition="'$(SkipFrameworkAssemblyMetadata)' != 'true' and '@(AssemblyMetadata)' != ''">
```

Comment:
Just curious, why are we removing the `SkipFrameworkAssemblyMetadata` condition?

Reply:
It doesn't appear to be used.

https://github.com/dotnet/runtime/search?q=SkipFrameworkAssemblyMetadata

Also, I'm assuming it is for the case where `IsTrimmable` should be generated for the assembly, but someone adds `SkipFrameworkAssemblyMetadata` to the project. That would block `IsTrimmable` from getting generated.

Reply:
Yup - see also the second commit message: https://github.com/dotnet/runtime/pull/48428/commits/d03327e62ddb7035442c9b803c699d1f8776e1bd